doc/nrf/ug_gzll.rst

```diff
@@ -106,32 +116,45 @@ Gazell automatically takes care of all synchronization and packet handling.
 You need to add payloads to the transmit (TX) FIFOs and read payloads from the receive (RX) FIFOs.
 Gazell automatically notifies the application when a packet is received.
 
-To set up a Gazell application, do the following:
+To set up a Gazell application, complete the following steps:
+
+1. Initialize Gazell Link Layer glue code using :c:func:`gzll_glue_init()`.
+#. Initialize Gazell using :c:func:`nrf_gzll_init()` and choose either Host or Device.
+#. Reconfigure Gazell's default parameters.
+   At a minimum, reconfigure the addresses and channels to avoid interfering with other Gazell networks.
```

Comment:
In the cases with additional info to a step, I would rather add an empty line so that the additional info is shown on a separate line between steps.